xenit-alfresco/templates/active-mq/mq-deployment.yaml

```diff
@@ -84,6 +84,16 @@ spec:
               mountPath: /opt/activemq/data
               subPath: mq/data
           {{- end }}
+          {{- if .Values.persistentStorage.mq.additionalClaims }}
```

Comment:
If you also want to enable addittionalClaims (to be managed by the helm helper) you should also alter the mq-volumes.yaml (see alfresco-volumes.yaml)
`{{- if .additionalClaims }}
{{- range .additionalClaims }}
{{- $name := .name -}}
{{- $storageClassName := .storageClassName -}}
{{- $storage := .storage -}}
{{- $efsVolumeHandle := .efs.volumeHandle -}}
{{- include "hepers.volumeHelper" (list $namespace $name $storageClassName $storage $efsVolumeHandle) }}`